modules/server_processes_manager/php/serverprocesslauncher.class.inc

```diff
@@ -36,17 +36,6 @@ class ServerProcessLauncher
      */
     private $_databaseProvider;
 
-    /**
-     * Builds a new ServerProcessesLauncher
-     *
-     * @param IDatabaseProvider $databaseProvider entity used to access the database
-     */
-    public function __construct(
-        IDatabaseProvider $databaseProvider
-    ) {
-        $this->setDatabaseProvider($databaseProvider);
-    }
-
```

Comment:
@nicolasbrossard requested your review for this part.. as far as I can tell, the only location where a ServerProcessesLauncher is instantiated there's no parameter passed.

Reply:
Since `$databaseProvider` is initialized (lazily) in the getter method, your change is legit. Approved.